### ISSUE TITLE
Rosetta is failing to pull localeInfo.json

### ISSUE THREAD

**Issue by mattpen** [phetsims/rosetta#237]
```
Jul 10 09:13:07 phet-server rosetta: 2020-07-10 09:13:07 | info - a request for locale info file is in progress, waiting on that promise
Jul 10 09:13:07 phet-server rosetta: 2020-07-10 09:13:07 | error - unhandled rejection, error.message = request to https://raw.githubusercontent.com/phetsims/chipper/master/data/localeInfo.json failed, reason: getaddrinfo ENOTFOUND raw.gi
Jul 10 09:13:07 phet-server rosetta: 2020-07-10 09:13:07 | error - stack = FetchError: request to https://raw.githubusercontent.com/phetsims/chipper/master/data/localeInfo.json failed, reason: getaddrinfo ENOTFOUND raw.githubusercontent.c
Jul 10 09:13:07 phet-server rosetta: at ClientRequest.<anonymous> (/data/share/phet/rosetta/node_modules/node-fetch/index.js:133:11)
Jul 10 09:13:07 phet-server rosetta: at emitOne (events.js:116:13)
Jul 10 09:13:07 phet-server rosetta: at ClientRequest.emit (events.js:211:7)
Jul 10 09:13:07 phet-server rosetta: at TLSSocket.socketErrorListener (_http_client.js:401:9)
Jul 10 09:13:07 phet-server rosetta: at emitOne (events.js:116:13)
Jul 10 09:13:07 phet-server rosetta: at TLSSocket.emit (events.js:211:7)
Jul 10 09:13:07 phet-server rosetta: at emitErrorNT (internal/streams/destroy.js:73:8)
Jul 10 09:13:07 phet-server rosetta: at _combinedTickCallback (internal/process/next_tick.js:139:11)
Jul 10 09:13:07 phet-server rosetta: at process._tickCallback (internal/process/next_tick.js:181:9)
```

There was a reboot of phet-server on Wednesday July 8.  Maybe some untested changes were unintentionally pulled and deployed.  There were also some problems with services not able to read from the filesystem, but I'm not seeing anything in the logs to indicate that is the problem here.

This is a fatal error and is causing rosetta to not respond when signed in, see https://github.com/phetsims/website/issues/1104.

@jbphet @muedli please investigate.

**Comment by jbphet** [comment]
Jumping in on this.  I just checked the log, and it looks like we're at SHA 3277ce6127c846d3a41ffad4499b0fb8ba2ccc90 from June 9 2020, which is basically where it should be, meaning that I don't think it's running untested code.

**Comment by jbphet** [comment]
I looked through the log and it's definitely having trouble getting files from GitHub.  I tried a `wget` command for the same resource, and it worked fine, so I just tried restarting Rosetta, and that seems to have done the trick - it's working fine now.  My guess is that the DNS service wasn't fully ready when the process was starting, since the startup was triggered by a full reboot of the server, and that left it in an odd state.  Not much else to do here - closing.
